File: tests/teststatsd.js
#!/bin/env node

var sdc_init = require('statsd-client')  
var sdc = new sdc_init({ host: 'localhost', port:8125 });  

var end = +new Date + 24 * 3600 * 1000; // 1 day 
var i = 0;
while(+new Date < end ) {
	i++;	
	sdc.increment('test.backend.nodejs.x1', (+new Date) % 100 );  
	// sdc.increment('test.backend.nodejs.x1', 5);  
	if( i % 10 == 0) {
        	console.log(i);	
		console.log(+new Date);
	}
	sleep(100);
}
sdc.close(); 

function sleep(sleepTime) {
    for(var start = +new Date; +new Date - start <= sleepTime; ) { } 
}

File: tests/teststatsd.py
#!/usr/bin/env python


import statsd
import os
import sys
import time
import math
import random

c=statsd.client.StatsClient(host='metric.server.com',port=8125)
#c=statsd.client.StatsClient(host='192.168.64.5',port=8125)

i = 0
while i < 10:
    # time.sleep(1)
    c.incr("dev.pandora.x1", random.choice(range(100)))
    i=i+1
    print i
	


print "done"
